app/bundles/UserBundle/Controller/UserController.php

```diff
@@ -136,10 +136,15 @@ public function newAction()
             if (!$cancelled = $this->isFormCancelled($form)) {
                 //check to see if the password needs to be rehashed
                 $formUser          = $this->request->request->get('user', []);
+                $automaticPassword = $formUser['automaticPassword'];
+
+                error_log('stampo il type di auto');
+                error_log(gettype($automaticPassword));
+                error_log($automaticPassword);
```

Comment:
These debug lines should not be part of this PR

Reply:
My distraction, i deleted these lines